api/auth.py

```diff
@@ -39,8 +39,9 @@ class Authentication:
     should be a db.Database object.
     """
 
+    CRYPTCTX = CryptContext(schemes=["bcrypt"], deprecated="auto")
```

Comment:
I'd suggest `CRYPT_CTX` for better readability.
```suggestion
    CRYPT_CTX = CryptContext(schemes=["bcrypt"], deprecated="auto")
```